### PR TITLE
Initial updates for gradle

### PR DESCRIPTION
**Pull request by mandy-chessell**
Signed-off-by: Mandy Chessell <mandy.e.chessell@gmail.com>

This PR has some preliminary updates for moding the developer dojo to gradle

### CODE REVIEW

File: site/docs/education/egeria-dojo/developer/develop-asset-listen-cut-paste.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project. -->
+
+??? beginner "Create a new IntelliJ project for AssetListen called `egeria-dojo1`"
+    ---8<-- "docs/education/tutorials/intellij-tutorial/intellij-new-project.md"
+
+??? beginner "Create a new *Sources Root* in `egeria-dojo1` under `src` called `main/java`"
```
Comment (planetf1) [inline site/docs/education/egeria-dojo/developer/develop-asset-listen-cut-paste.md:7]: Normally sources root gets set automatically when a maven pom (or gradle build file) is imported. However in this sequence you are creating package/class before the pom (quite reasonably) so it's needed sooner. 

One alternative is to create the pom first. Just mentioning for considerations as it removes steps, these instructions are still fine

File: site/docs/education/egeria-dojo/developer/different-types-of-apis-platform-deployment.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ In `egeria-dev-projects`, there is a module called `egeria-platform-report`.  Th
               Asset Owner OMAS: Manage an asset
               Stewardship Action OMAS: Manage exceptions and actions from open governance
               Governance Program OMAS: Manage the governance program
-              Digital Service OMAS: Manage a digital service's lifecycle
+              Digital Service OMAS: Manage a digital service through its lifecycle
```
Comment (planetf1) [inline site/docs/education/egeria-dojo/developer/different-types-of-apis-platform-deployment.md:106]: it's ?

File: site/docs/education/egeria-dojo/developer/overview.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ This dojo was last tested using Egeria release 3.12.
 
     !!! attention "Think about your machine type ..."
 
-        The instructions below cover fairly standard machine architectures and set ups.  If you are using, say the new M1 chip in your Mac, or are running a work machine that has been locked down by your company, then you may need to take additional actions, install additional software and enable additional permissions. In all cases, check that the software being installed is correct for your machine type and its approved uses.
+        The instructions below cover fairly standard machine architectures and set ups.  If you are using, say the new M1 chip in your Mac, or Windows, or are running a work machine that has been locked down by your company, then you may need to take additional actions, install additional software and enable additional permissions. In all cases, check that the software being installed is correct for your machine type and its approved uses.
```
Comment (planetf1) [inline site/docs/education/egeria-dojo/developer/overview.md:32]: I suspect we can remove m1 reference now as enough of the team are using m1 just fine (the only real issue we had was with architecture specific binaries -- that was only containers).

The issue with windows remains

File: site/docs/education/tutorials/building-egeria-tutorial/task-building-with-gradle.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+=== "Flexible platform option (egeria.git only)"
+    This build option creates an [OMAG Server Platform](/concepts/omag-server-platform) where the [registered services](/concepts/omag-subsystem/#registered-services) are optional.  The OMAG Server Platform loads the registered services it finds on the loader path specified with the `-Dloader.path={directoryName}` option of its startup command.  Use this option if you want to remove the registered services that you are not using, or you would like introduce your own registered services.
+
+    Note: this feature is still in development - see issue [#7309](https://github.com/odpi/egeria/issues/7309) - use Maven command `mvn clean install -DadminChassisOnly` instead.
+    ```shell
+    ./gradlew build ???
+    ```
+
+The build can take 15 minutes to over an hour depending on the repository and on the speed/load on your machine.  However eventually you will see the message:
```
Comment (planetf1) [inline site/docs/education/tutorials/building-egeria-tutorial/task-building-with-gradle.md:37]: seconds to  minutes for the compile, and minutes-10 minutes for tests (as this is gradle section)

File: site/docs/education/tutorials/building-egeria-tutorial/terminal-installing-egeria.md [hunk 5]
```diff
@@ -16,10 +16,10 @@ The name of the files is determined by the release level of the code that you [d
 ```bash
 antrun                                                  egeria-3.12-SNAPSHOT-distribution
 archive-tmp                                             egeria-3.12-SNAPSHOT-distribution.tar.gz
-egeria-3.12-SNAPSHOT-deploy                              open-metadata-assemblies-3.12-SNAPSHOT-javadoc.jar
-egeria-3.12-SNAPSHOT-deploy.tar.gz                       open-metadata-assemblies-3.12-SNAPSHOT-sources.jar
+egeria-3.12-SNAPSHOT-deploy                             open-metadata-assemblies-3.12-SNAPSHOT-javadoc.jar
+egeria-3.12-SNAPSHOT-deploy.tar.gz                      open-metadata-assemblies-3.12-SNAPSHOT-sources.jar
 ```
-The install image tar file is `{{release}}-distribution.tar.gz` or `egeria-3.12-SNAPSHOT-distribution.tar.gz` in this example.
+The distribution tar file is `{{release}}-distribution.tar.gz` or `egeria-3.12-SNAPSHOT-distribution.tar.gz` in this example.
```
Comment (planetf1) [inline site/docs/education/tutorials/building-egeria-tutorial/terminal-installing-egeria.md:22]: note that for the gradle build, the distribution is in a slightly different location to maven
```
./open-metadata-distribution/open-metadata-assemblies/build/   unpacked/egeria-${RELEASE}-distribution.tar.gz/server/${CHASSIS}-${RELEASE}.jar
```

**Review by planetf1 — COMMENTED** [review]
Just some minor/very minor comments that may be useful info/suggestions

File: site/docs/education/egeria-dojo/developer/develop-asset-listen-cut-paste.md (hunk 1 above)
Comment (mandy-chessell) [reply site/docs/education/egeria-dojo/developer/develop-asset-listen-cut-paste.md:7]: The aim is to help people understand how intellij works

File: site/docs/education/egeria-dojo/developer/different-types-of-apis-platform-deployment.md (hunk 2 above)
Comment (mandy-chessell) [reply site/docs/education/egeria-dojo/developer/different-types-of-apis-platform-deployment.md:106]: The apostophe is a problem to the formatting

File: site/docs/education/tutorials/building-egeria-tutorial/terminal-installing-egeria.md (hunk 5 above)
Comment (mandy-chessell) [reply site/docs/education/tutorials/building-egeria-tutorial/terminal-installing-egeria.md:22]: The whole purpose for updating this section is to change the location of the distribution and the instructions we have to assume a gradle build - the location shown is for the gradle build - not maven.

**Pull request merged**